packages/bot-web-ui/src/components/transaction.jsx

```diff
@@ -130,6 +130,7 @@ const Transaction = ({
     setActiveTransactionId,
 }) => (
     <Popover
+        zIndex = {10}
```

Comment:
Should be passed as a constant.

Reply:
@aaron-binary we don't have any constants for zindex.

Reply:
We do now. 👍